library/certificate_request.py

```diff
@@ -184,6 +246,10 @@ def _get_argument_spec():
             owner=dict(type="str"),
             group=dict(type="str"),
             principal=dict(type="list"),
+            key_usage=dict(
```

Comment:
If I wanted to create a cert with no key_usage and no extended_key_usage, would I do this?
```yaml
key_usage: []
extended_key_usage: []
```
?

Reply:
You can but with some limitations. Certmonger can issue certificates without `keu_usage` and `extended_key_usage` but it cannot remove all `key_usage` from a certificate.

While this is a limitation in Certmonger it's not really a big problem because there is not much use for a certificate with no `key_usage`. 

@rcritten does that sound right?

Reply:
Agreed. I don't know if it is even valid per the RFC(s) to have no key_usage but it would be a very silly thing to do. I don't know what the use-case would be.